### PR TITLE
South Africa_1-Ryan_Smith-JavaScript-Week1

### PR DESCRIPTION
**Pull request by RyanS-Cpt**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: exercises/G-numbers/exercise.js [hunk 1]
```diff
@@ -1 +1,8 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 22;
+var numberOfMentors = 15;
+
+sum = numberOfStudents + numberOfMentors;
```
Comment (sinopungula) [inline exercises/G-numbers/exercise.js:5]: Hey, love the efficiency of using sum instead of creating another var.

**Review by sinopungula — COMMENTED** [review]
Hey Ryan! Reviewed your work, great job.

**Comment by RodrigueNdzana** [comment]
Hi Ryan, hope your are doing well. Well done.

**Review by theTrainMan — COMMENTED** [review]
Hey Ryan, looking good. We must arrange a session together.

File: exercises/G-numbers/exercise.js (hunk 1 above)
Comment (RyanS-Cpt) [reply exercises/G-numbers/exercise.js:5]: Thanks Sino.

**Pull request closed without merging**